domains/shivam27

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "shiv1819",
+    "email": "shivampanchal11q@gmail.com"
+  },
+  "record": {
+    "TXT": "79ac9ab8571f3432e2ba77d0709fa9"
```

Comment:
TXT doesn't do anything. Was this by any chance a Github Pages TXT verification? Because if so thats not how you do it.